src/providers/python.rs

```diff
@@ -301,6 +316,26 @@ impl PythonProvider {
             .map(|m| m.get(2).unwrap().as_str().to_string()))
     }
 
+    fn parse_tool_versions_python_version(file_content: &str) -> Result<Option<String>> {
+        let asdf_versions = parse_tool_versions_content(file_content);
+
+        // the python version can only specify a major.minor version right now, and not a patch version
+        Ok(asdf_versions.get("python").map(|s| {
+            let parts: Vec<&str> = s.split('.').collect();
+            assert_eq!(
```

Comment:
Instead of asserting here, could we instead just use the major/minor versions and warn that specifying the exact patch version is not supported